accumulator/forest.go

```diff
@@ -113,39 +113,40 @@ type Forest struct {
 //               forest types though (meaning there isn't functionality implemented
 //               to convert a CowForest to DiskForest and vise-versa). Pass a filepath
 //               and cowMaxCache(how much MB to use in ram) to create a CowForest.
-func NewForest(forestFile *os.File, cached bool,
-	cowPath string, cowMaxCache int) *Forest {
+
+type ForestName int
```

Comment:
ForestName isn't quite right. There are different types of forests so this should be forestType